core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -118,6 +117,49 @@ describe('Overview', () => {
             expect(jazzServices_po.getFunctionType().getText()).toEqual('function');
             expect(jazzServices_po.getFunctionStatus().getText()).toEqual('creation started');
       });
+
+      it('Verify Lamda Function Page Title', () => {
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            jazzServices_po.getLamdaName().click();
+            expect(jazzServices_po.getOverviewStatus().getText()).toEqual('OVERVIEW');
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(45000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(30000);
```

Comment:
don't you think you can add something like afterEach where you can put a timer instead along with other cleanup stuff (that needs to be run after every test)?

Reply:
what the purpose of Refresh here is we have one button in UI that needs to be clicked inorder to get the Prod and Stg